amplify/backend/api/householdapp/schema.graphql

```diff
@@ -96,6 +96,7 @@ type EventHandler
   calendarId: ID! @index(name: "byCalendar")
   Calendar: Calendar @belongsTo(fields: ["calendarId"])
   Events: [Event] @hasMany(indexName: "byEventHandler", fields: ["id"])
+  taskId: ID! @index(name: "byTask")
```

Comment:
**curious**: why are we adding an index here?
A Task and an EventHandler have a 1..1 connection. Meaning they both have a field that contains each other's ID, we should not need to query the whole EventHandler table.

Reply:
I think I added that index by mistake. I never used it, so I assume I just added it subconsciously. I'll remove it thanks.